modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -653,38 +659,38 @@ public void testAddObjectEntryWithFormulaObjectField() throws Exception {
 
 	@Test
 	public void testAddObjectEntryWithObjectValidationRule() throws Exception {
+		List<Map<String, String>> expectedObjectValidationRuleResults =
+			new ArrayList<>();
```

Comment:
I changed it to `List<ObjectValidationRuleResult>`, but calling `Assert.assertArrayEquals` didn't work because it compares the references of the `ObjectValidationRuleResult` instances, which will be different. So I maintained the for loop. Does it make sense?

Reply:
@devpedrotavares I was thinking doing something like [this](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectStateFlowLocalServiceTest.java#L158-L166).

Reply:
We talked and opted to keep the for loop